go-backend/migrations/1587467112_create_organisation_table.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE organizations (
+  id serial PRIMARY KEY,
+	name varchar(50),
+	contact_email varchar(50),
+	domain_name varchar(45),
+	subscription_status integer,
+	subscription_valid_upto timestamp,
+	hi5_limit integer,
+	hi5_quota_renewal_frequency varchar(50),
+	timezone varchar(100),
+	created_by integer,
+	created_on timestamp DEFAULT CURRENT_TIMESTAMP,
+	updated_by integer,
+	updated_on timestamp DEFAULT CURRENT_TIMESTAMP
```

Comment:
@mayuriardad can you make sure that if we need a timestamp here or we need to store epoch time here. Pls check this with node team